src/marker_on_image.cpp

```diff
@@ -127,7 +127,9 @@ class MarkerOnImage
 
       // Remove if not supported
       if (it->msg.type != visualization_msgs::Marker::SPHERE &&
```

Comment:
Would it be easier to have a const set of supported messages?

Reply:
Something like this? https://github.com/osrf/marker_on_image/pull/2/commits/16a12b9ec34b327a66912a20749a204868f3c8be